open_academy/models/course.py

```diff
@@ -9,3 +9,11 @@ class Course(models.Model):
     description = fields.Text()
     responsible_id = fields.Many2one("res.users", "Responsible")
     session_ids = fields.One2many("session", "course_id")
+
+    def copy(self):
+        default = {}
+        default.update({
```

Comment:
you can declare the dict and assign it

default = {"title": f"Copy of {self.title}"}

Reply:
Done